app/navigation/navigation-core/src/main/kotlin/com/hedvig/android/navigation/core/AppDestination.kt

```diff
@@ -54,6 +54,14 @@ sealed interface AppDestination : Destination {
     val insuranceDisplayName: String,
   ) : AppDestination
 
+  @Serializable
+  data class InsuranceContractDetail(
+    val contractId: String,
+  ) : AppDestination
+
+  @Serializable
+  data object TerminatedInsurances : AppDestination
```

Comment:
Same comment as above, I think we can avoid moving these destinations to the global AppDestination interface, and keep them only to the relevant module destinations.